server/api/hot_status.go

```diff
@@ -38,6 +41,27 @@ type HotStoreStats struct {
 	QueryReadStats  map[uint64]float64 `json:"query-read-rate,omitempty"`
 }
 
+// HistoryHotRegionsRequest wrap request condition from tidb.
+// it is request from tidb
+type HistoryHotRegionsRequest struct {
+	StartTime int64    `json:"start_time,omitempty"`
+	EndTime   int64    `json:"end_time,omitempty"`
+	RegionIDs []uint64 `json:"region_ids,omitempty"`
+	StoreIDs  []uint64 `json:"store_ids,omitempty"`
+	PeerIDs   []uint64 `json:"peer_ids,omitempty"`
+	//0 means not leader,1 means leader
+	Roles          []int64  `json:"is_leader,omitempy"`
```

Comment:
Use bool and change the variable name?

Reply:
In TiDB,bool use int64

Reply:
BTW, it's better to be consistent with the JSON tag.